### ISSUE TITLE
Improve Documentation for Using SVG 

### ISSUE THREAD

**Issue by telmiger** [Jermolene/TiddlyWiki5#4417]
Following up on a [discussion in the Goolge Group](https://groups.google.com/d/msg/tiddlywiki/Vh-LXr3VdeM/yM1hZgaiEQAJ) I re-read the docs about [Using SVG](https://tiddlywiki.com/#Using%20SVG). Issues:

- introduction at the top for the modes **image** vs. **inline** or **embedded** (which term would be best for the section title?) might need more explanation or better structure: the first paragraph ends with "See below for an example.", that should point to the embed example, but below follows the image example.
- example **Motovun Jack.svg** is a missing tiddler
- embedding example {{Motovun Jack.jpg}} is not an svg
- example with WikiText on blue background (circle) is hardly readable

Adding information about importing could be valuable, as well as to point out that thanks to tiny file sizes of SVG images, embedding usually would be better (performance) or at least no problem compared to using external images and other image formats.

The whole tiddler is rather long and might be devided into two:

1. Using SVG
-- Image container for tiddlers with type image/svg+xml
-- Inline SVG and WikiText
-- Importing SVG vs. [external images](https://tiddlywiki.com/#ExternalImages)
2. Using SVG (Examples)

**Comment by TiddlyTweeter** [comment]
I want to underline Thomas' points with stress on clarifying the TWO types of SVG. A common confusion is that users are often getting in a mess not grasping the difference between SVG HTML elements (no xmls declaration) and SVG "files" in TW. The confusion is that we can hold SVG image "files" in TW and SVG elements. They look very similar, but are quite different in the way you need to handle them.

**Comment by Jermolene** [comment]
Hi @telmiger @TiddlyTweeter improvements to the SVG docs are very welcome.

(I think "Motovon Jack.svg" is not a missing a tiddler, but perhaps I looked in the wrong place?)

**Comment by telmiger** [comment]
> I think "Motovon Jack.svg" is not a missing a tiddler,
You are right, @Jermolene, must have been the version of tiddlywiki.com in my browser cache.

**Comment by TiddlyTweeter** [comment]
> Hi @telmiger @TiddlyTweeter improvements to the SVG docs are very welcome.

I not sure on all @telmiger wants.

The thing I felt was central was simply emphasizing more the difference between SVG HTML elements and SVG "files" (embeds in TW). It seems to be that difference that is confusing users most?

**Comment by AnthonyMuscio** [comment]
Being able to colour an SVG icon from the tiddler that embeds it, and display it at different sizes are common user requirements. Fool proof instructions (including for me) for these basic needs, 

How we handle image sizes need more info which may as well be the same thing as SVG's far as a user cares.
